tools/clang/test/CodeGenHLSL/linalg/outer-product-accumulate-matrix-layout.hlsl

```diff
@@ -0,0 +1,65 @@
+// RUN: %dxc -T cs_6_9 %s -enable-16bit-types -DML=RowMajor -DSTRIDE=64 2>&1| FileCheck %s --check-prefixes DXIL-0
+// RUN: %dxc -T cs_6_9 %s -enable-16bit-types -DML=ColumnMajor -DSTRIDE=64 2>&1 | FileCheck %s --check-prefixes DXIL-1
+// RUN: %dxc -T cs_6_9 %s -enable-16bit-types -DML=MulOptimal -DSTRIDE=64 2>&1 | FileCheck %s --check-prefixes DXIL-2
+// RUN: %dxc -T cs_6_9 %s -enable-16bit-types -DML=OuterProductOptimal -DSTRIDE=64 2>&1 | FileCheck %s --check-prefixes DXIL-3
+// RUN: %dxc -T cs_6_9 %s -enable-16bit-types -DML=OuterProductOptimal -DSTRIDE=0 2>&1 | FileCheck %s --check-prefixes DXIL-4
+ByteAddressBuffer input_vector_buffer;
+ByteAddressBuffer input_vector_buffer2;
+RWByteAddressBuffer matrix_buffer;
+
+enum CompType {
```

Comment:
Addressing comment https://github.com/microsoft/DirectXShaderCompiler/pull/7412#discussion_r2070712329

I followed the pattern in `outer-product-accumulate-multioverload.hlsl` for consistency. I am fine changing to your suggestion if you prefer it.

Reply:
These tests predated the high-level API header.  I think we can switch to defined values from the high-level API now.